src/loaders/assetLoader.js

```diff
@@ -111,18 +115,20 @@ module.exports = async function assetLoader() {
                   throw new Error(`Unknown scale ${scale} for ${filepath}`);
               }
 
+              const longname = `${`${url.replace(/\//g, '_')}_${filename}`
```

Comment:
Just wanted to scope it for now. Ultimately I want it to go to a separate function, so that will be gone.